src/model/request/loginRequest.ts

```diff
@@ -1,3 +1,22 @@
-export class LoginRequest {
-    constructor(public username?: string, public password?: string){ }
+import { BaseRequest } from "./BaseRequest";
+import { MessageField } from "../utils/messageField";
+
+export class LoginRequest extends BaseRequest{
+    constructor(public username?: string, public password?: string, public detail?: Array<MessageField>){
+        super();
+    };
+
+    public validate(): boolean {
```

Comment:
Nice oo programming thank you for thinking ahead, 
However
1 these modules are publically published so want to avoid putting any business logic in here 
2 this logic would be much better fit in this system in a service static class in the fr8back project.
Since no other systems other than the fr8back system needs to know *how* to validify a logo request

Reply:
ok i will put the validate() method into userService.ts in Fr8back. is it ok for you?

Reply:
yup it will awesome Jimmy :)